### DISCUSSION TITLE
Upgrade to jackson 2.15.x

### DISCUSSION THREAD

**Discussion by pmancaux** [logfellow/logstash-logback-encoder#958] [Q&A] [answered]
Hello, 

do you plan to migrate to last jackson version 2.15.0?

I'm using logstash-logback-encoder in my application that use 2.15.0 of jackson and when i'm starting under tomcat10 with logstas-logback-encoder-7.3 i have this error : 

```
0:47:23,717 |-ERROR in net.logstash.logback.encoder.LoggingEventCompositeJsonEncoder@161ef09f - Error occurred while dynamically loading jackson modules java.util.ServiceConfigurationError: com.fasterxml.jackson.databind.Module: com.fasterxml.jackson.datatype.jsr310.JavaTimeModule not a subtype
	at java.util.ServiceConfigurationError: com.fasterxml.jackson.databind.Module: com.fasterxml.jackson.datatype.jsr310.JavaTimeModule not a subtype
	at 	at java.base/java.util.ServiceLoader.fail(ServiceLoader.java:593)
```

perhaps its due to the difference of jackson version reference by logstah (2.14.2) and in my project 2.15.0?

**EDIT1**

=> same for logback-* dep, 7.3 use logback-* 1.3.x but last version is 1.4.x to be jakarta.* compatible?

Thanks for help

**Comment by tomitoth** [comment]
Hi there,
I would also be interested in if there is a plan to move towards the jakarata namespace compatibility by updating the logback dependency to 1.4.x. We plan to update to Spring Boot 3 as 2.7 is closing to is end of life. In Spring Boot 3 they started to use Jakarta EE 9.
Thanks

**Reply by philsttr** [reply, marked as answer]
@pmancaux Regarding Jackson 2.15.... Yes, dependabot manages updates to all the dependencies of this project.  See #964.

@tomitoth regarding Java EE -> Jakarta EE... logstash-logback-encoder does not use any Java EE apis.  You can use dependencyManagement in your project to manage the logback dependencies up to 1.4.x, and logstash-logback-encoder 7.3 will work properly with logback 1.4.
